system_test/helpers/aest_docker.erl

```diff
@@ -101,6 +105,29 @@ stop(BackendState) ->
     log(BackendState, "Networks pruned", []),
     ok.
 
+-spec prepare_spec(node_spec(), backend_state()) -> node_spec().
+prepare_spec(#{pubkey := PubKey, privkey := PrivKey} = Spec, BackendState) ->
+    #{data_dir := DataDir} = BackendState,
+    #{name := Name} = Spec,
+    KeysDir = keys_dir(DataDir, Name),
+    Password = ?PEER_KEYS_PASSWORD,
+    {PubFile, PrivFile} = aec_keys:peer_key_filenames(KeysDir),
+    file:delete(PubFile),
+    file:delete(PrivFile),
+    aec_keys:save_peer_keys(Password, KeysDir, PubKey, PrivKey),
```

Comment:
I don't understand this. Would you not need to save these keys on the container? It looks like you are saving them on the local disk, since aest_nodes calls this function from outside the container.

Puzzled

Reply:
Because later on this directory is mounted inside the container.